src/ethereum_test/blockchain_test.py

```diff
@@ -22,6 +22,11 @@
     JSONEncoder,
 )
 
+default_base_fee = 1
```

Comment:
This should be 7 because it is impossible for the base fee to go below that value.